src/esp/sensor/Sensor.h

```diff
@@ -44,22 +44,12 @@ enum class SensorSubType {
 
 // Specifies the configuration parameters of a sensor
 struct SensorSpec {
-  std::string uuid = "rgba_camera";
-  SensorType sensorType = SensorType::Color;
-  SensorSubType sensorSubType = SensorSubType::Pinhole;
-  std::map<std::string, std::string> parameters = {{"near", "0.01"},
-                                                   {"far", "1000"},
-                                                   {"hfov", "90"},
-                                                   {"ortho_scale", ".1"}};
+  SensorSpec() {}
+  std::string uuid;
   vec3f position = {0, 1.5, 0};
   vec3f orientation = {0, 0, 0};
-  vec2i resolution = {84, 84};
-  int channels = 4;
-  std::string encoding = "rgba_uint8";
-  // description of Sensor observation space as gym.spaces.Dict()
-  std::string observationSpace = "";
   std::string noiseModel = "None";
```

Comment:
I kept noiseModel in SensorSpec because the python bindings had many methods which referred to the noiseModel. From is_valid_sensor_type, it seems like only VisualSensors have noiseModels which are not "None" (but can be). NonVisualSensors can only have noiseModel = "None", so I may move this to VisualSensor. Will need to look more into python bindings @Skylion007 let me know if you have any thoughts!

Reply:
You do not have to give it a name `None`, you can just keep the string empty. 
And keep this term in the SensorSpec in case non-visual sensor also need it. That is it.